src/app/public/modules/index.ts

```diff
@@ -1,11 +1,16 @@
 export * from './adapter-service';
 
+export * from './affix/affix-auto-fit-context';
 export * from './affix/affix-config';
 export * from './affix/affix-horizontal-alignment';
+export * from './affix/affix-offset-change';
+export * from './affix/affix-offset';
 export * from './affix/affix-placement';
 export * from './affix/affix-placement-change';
 export * from './affix/affix-vertical-alignment';
 export * from './affix/affix.module';
+export * from './affix/affix.service';
```

Comment:
Are consumers supposed to use the service? Or should they only ever be using the directive?

Another (maybe unrelated) question - I noticed the docs page has info for the `createAffixer()` and it links to `SkyAffixer` (but the link is broken). Is that intended to also be exposed to consumers?

Reply:
Both should be available to consumers (though, they should only be used internal to our components for now). I have an issue to allow docs-tools to document plain old TypeScript classes, here: https://github.com/blackbaud/skyux-docs-tools/issues/30